plugins/com.google.cloud.tools.eclipse.appengine.libraries/schema/com.google.cloud.tools.eclipse.appengine.libraries.exsd

```diff
@@ -18,177 +18,226 @@
       </annotation>
       <complexType>
          <choice minOccurs="1" maxOccurs="unbounded">
-            <element ref="library"/>
+            <element ref="library" />
          </choice>
-         <attribute name="point" type="string" use="required">
-            <annotation>
-               <documentation>
-                  
-               </documentation>
-            </annotation>
-         </attribute>
-         <attribute name="id" type="string">
-            <annotation>
-               <documentation>
-                  
-               </documentation>
-            </annotation>
-         </attribute>
+         <attribute name="point" type="string" use="required" />
+         <attribute name="id" type="string" />
          <attribute name="name" type="string">
             <annotation>
-               <documentation>
-                  
-               </documentation>
                <appinfo>
-                  <meta.attribute translatable="true"/>
+                  <meta.attribute translatable="true" />
                </appinfo>
             </annotation>
          </attribute>
       </complexType>
    </element>
 
    <element name="library">
+      <annotation>
+         <documentation>
+            An App Engine library that can consist of one or more jar files with associated visibility rules and Maven coordinates.
+         </documentation>
+      </annotation>
       <complexType>
          <choice minOccurs="1" maxOccurs="unbounded">
-            <element ref="jarFile"/>
+            <element ref="libraryFile" />
          </choice>
          <attribute name="id" type="string">
             <annotation>
                <documentation>
-                  
+                  The ID of the library. The path of the classpath container defined for this library will be 
+'com.google.cloud.tools.eclipse.appengine.libraries/ID' where ID is the value defined in this attribute. 
+The classpathContainerInitializer defined to resolve the container will have the ID 
+'com.google.cloud.tools.eclipse.appengine.libraries'. The second section of the container path (the ID after the '/') 
+will determine how to resolve the entries associated with the container.
                </documentation>
             </annotation>
          </attribute>
          <attribute name="name" type="string">
             <annotation>
                <documentation>
-                  
+                  A user-friendly name for the library that can be displayed in the UI, e.g.: &quot;AppEngine Endpoints&quot;.
```

Comment:
You don't actually need to use &quot; here. Literal " works just fine and i a tad more legible.